base/BUILD

```diff
@@ -113,73 +113,73 @@ go_binary(
 )
 
 # Replicate the containers and tests for debian9 and debian10
-distro_components("-debian9")
+distro_components("_debian9")
 
-distro_components("-debian10")
+distro_components("_debian10")
 
 # alias debian9 as the default images
 alias(
     name = "static",
-    actual = ":static-debian9",
+    actual = ":static_debian9",
```

Comment:
Do these aliases refer to Docker image names or Bazel target names?

Reply:
These are Bazel aliases, which are there to ensure that if anyone is depending on these images from native Bazel rules, they don't need to update: `//base:static` is now an alias for `//base:static_debian9`. They are also used by the packages in this repository that have not been updated to be distribution-aware yet (Node, dotnet, Java).